observations/obs_converters/GSI2DART/gsi_to_dart.f90

```diff
@@ -53,7 +53,12 @@ program gsi_to_dart
 call mpi_initialize
 
 ! Print out some info
-call initialize_mpi_utilities('gsi_to_dart',communicator=my_communicator)
+call initialize_mpi_utilities('gsi_to_dart',communicator=mpi_comm_world)
```

Comment:
i'd leave the original unless there's a good reason not to?

Reply:
This is odd ... the 'original' called initialize_utilities(), I changed it to call initialize_mpi_utilities().
I think you're saying you like using 'my_communicator' better.